open-metadata-implementation/access-services/subject-area/subject-area-client/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/client/SubjectAreaEntityClient.java

```diff
@@ -11,14 +11,39 @@
 
 import java.util.List;
 
+/**
+ * @param <E> inherited from {@link Node} object type
+ * Interface for Subject Area client entity operations
+ */
 public interface SubjectAreaEntityClient<E extends Node> extends SubjectAreaClient<E> {
 
+    /**
+     * Request to search relationships for current entity.
```

Comment:
entity => node. ThisAPI doesn ot expose the concept of entity

Reply:
Okay.